src/braket/circuits/gate.py

```diff
@@ -40,6 +43,27 @@ def __init__(self, qubit_count: Optional[int], ascii_symbols: Sequence[str]):
                 `ascii_symbols` length != `qubit_count`
         """
         super().__init__(qubit_count=qubit_count, ascii_symbols=ascii_symbols)
+        self._adjoint = False
+
+    def adjoint(self) -> Gate:
+        """Returns the adjoint of this gate.
+
+        Returns:
+            Gate: The adjoint of this gate.
+        """
+        new = copy.copy(self)
+        new._adjoint = not self._adjoint
+        return new
+
+    def adjoint_list(self) -> List[Gate]:
+        """Returns a list of gates that comprise the adjoint of this gate.
+
+        For compatibility, it is best if all gates in ths list are of the same type as this gate.
+
+        Returns:
+            List[Gate]: The gates comprising the adjoint of this gate.
+        """
+        raise NotImplementedError(f"Gate {self.name} does not have adjoint implemented")
```

Comment:
I'm confused why this isn't:
`` return [self.adjoint()]

Reply:
See comment about difference between `adjoint` and `adjoint_list`